packages/wonder-blocks-dropdown/src/components/dropdown-core.tsx

```diff
@@ -544,20 +542,43 @@ class DropdownCore extends React.Component<Props, State> {
                 );
             }
 
-            const node = ReactDOM.findDOMNode(
-                focusedItemRef.ref.current,
-            ) as HTMLElement;
-            if (node) {
-                node.focus();
-                // Keep track of the original index of the newly focused item.
-                // To be used if the set of focusable items in the menu changes
-                this.focusedOriginalIndex = focusedItemRef.originalIndex;
-
-                if (onFocus) {
-                    // Call the callback with the node that was focused.
-                    onFocus(node);
+            const focusNode = () => {
+                // No point in doing work if we're not open.
+                if (!this.props.open) {
+                    return;
                 }
-            }
```

Comment:
**note:** This is unchanged, but is now called within a function so that we can delay it as needed if the virtualized DOM node isn't available.